sys/fuchsia/fidlgen/main.go

```diff
@@ -157,7 +110,7 @@ func fidlgen(fidlgenPath string, jsonPath string, txtPathBase string) string {
 		failf("fidlgen failed: %v", err)
 	}
 
-	return fmt.Sprintf("%s.txt", txtPathBase)
+	return fmt.Sprintf("%s.syz.txt", txtPathBase)
```

Comment:
What's the motivation for this change?

Reply:
We recently changed the fidlgen syzcaller backend to emit .syz.txt files to be more recognizable, this simply adjusts the handling. See https://fuchsia.googlesource.com/fuchsia/+/2494b080edbbc9e43b23180d18fe6f34ee4d6544

Reply:
ack